e2eshark/run.py

```diff
@@ -1117,6 +1305,16 @@ def main():
         "--skiptestsfile",
         help="A file with lists of tests that should be skipped",
     )
+    parser.add_argument(
+        "--upload",
+        help="Upload to azure storage based on uploadtestsfile",
+        action="store_true",
+        default=False,
+    )
+    parser.add_argument(
+        "--uploadtestsfile",
+        help="A file with lists of tests that should be uploaded",
+    )
```

Comment:
Looks like --uploadtestsfile is needed everytime --upload is used? If so, it would make sense to just have switch. You can get rid of --upload switch. If --uploadtestsfile is used, you have upload on.

Reply:
Also, please indicate to user where the files have been backed up and how to get it.

Reply:
Makes sense, did both